### ISSUE TITLE
[Web] [Accessibility] [11yMAS][Name, Role and Value] [MAS4.1.2][HighImpact][Role is not defined for all interactive controls in the Welcome screen.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#703]
**User Experience:**
User who rely on AT will not get proper information about the elements, if role is not defined which provides the purpose of control.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
OS Build: Mac OS Sierra 0.12.6
Browser: Safari 11.0.1,IE11
Tools used: Developer tool

**Attachments:**
![mas4 1 2_welcome screen](https://user-images.githubusercontent.com/33054846/32547182-61a03e74-c4a7-11e7-8406-14cc6221c398.png)

**Repro Steps:**
1.	Open the URL: https://script-lab.azureedge.net/.
2.	Navigate to main content and open developer tool.
3.	Verify Role is defined for all the Interactive controls in welcome screen or not.

**Current Behavior:**
Role is not defined for all the Interactive controls in welcome screen. Screen reader is announcing as "Group" for all controls. 

**Expected Behavior:**
Role should be defined as link for all the controls in welcome screen which helps the user to get to know the purpose of control. 

**MAS Reference**:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=4.1.2
